tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -853,3 +853,57 @@ def _extract_predictions(self, outputs_list, meta):
                 total_boxes = np.concatenate((total_boxes, boxes), axis=0)
 
         return [total_boxes]
+
+
+class FCOSPersonAdapter(Adapter):
+    """
+    Class for converting output of FCOS model to DetectionPrediction representation
+    """
+    __provider__ = 'fcos_person'
+    prediction_types = (DetectionPrediction, )
+
+    def validate_config(self):
+        super().validate_config(on_extra_argument=ConfigValidator.ERROR_ON_EXTRA_ARGUMENT)
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'output_blob': StringField(optional=True, default=None, description="Output blob name."),
+            'scale': NumberField(optional=True, default=1.0, description="Scale factor for bboxes."),
+        })
+
+        return parameters
+
+    def configure(self):
+        out_blob_name = self.get_value_from_config('output_blob')
+        if out_blob_name is not None:
+            self.out_blob_name = out_blob_name
+        else:
+            self.out_blob_name = self.output_blob
```

Comment:
please move this check in process, at the configuration stage `output_blob` can be not available yet. (e.g. quantization tools uses postponed model loading)
look how it is made in bert_classification adapter https://github.com/opencv/open_model_zoo/blob/develop/tools/accuracy_checker/accuracy_checker/adapters/nlp.py#L113-L117

Reply:
Thanks for the example. Fixed.